pl.c

```diff
@@ -196,7 +196,7 @@ static struct playlist *pl_new(const char *name)
 
 static void pl_free(struct playlist *pl)
 {
-	editable_clear(&pl->editable);
+	editable_clear(&pl->editable, 0);
```

Comment:
What if a track is in multiple playlists?

Reply:
Then it would presumably be removed prematurely, if it's to be consistent with what happens iif the track is in multiple playlists and gets individually removed from one playlist. That's the can of worms.